### ISSUE TITLE
URL are no longer highlighted since vte>=0.60

### ISSUE THREAD

**Issue by ncmans** [mate-desktop/mate-terminal#333]
#### Expected behaviour

URLs must be highlighted and when right clicked, there should be an option to copy the link or open it in browser.

#### Actual behaviour

URLs are treated like the rest of the content in the terminal.

#### Steps to reproduce the behaviour

1. Open `mate-terminal`
2. Type `echo https://mate-desktop.org` and press enter.
3. Hover over the URL

#### MATE general version

```
$ mate-terminal --version
MATE Terminal 1.24.0
```

#### Package version

```
$ pacman -Qi mate-terminal
Name            : mate-terminal
Version         : 1.24.0-1
...
```

#### Linux Distribution

Archlinux

#### Link to downstream report of your Distribution

Related bugs affecting other terminals:

* https://sourceforge.net/p/lxde/bugs/932/
* https://bugs.archlinux.org/task/65783
* https://github.com/JAugusto42/terminator/issues/3

**Comment by sc0w** [comment]
it was already fixed with https://github.com/mate-desktop/mate-terminal/commit/c21a7b374f420b224c24307325061b3bb55aa3bf

**Comment by gtnuux** [comment]
Maybe leave one of these reports open until the fix is released in, I assume, 1.24.1.  That would make it easier faster for people to find when they are trying to figure out what is going on.

Also, since this is a fairly significant annoyance, maybe it could be a reason to release an update sooner rather than later, if possible.

Thanks!
-Jeff
